src/accounts/accounts.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Model, Table } from 'sequelize-typescript';
+
+@Table
+export class Accounts extends Model<Accounts> {
+  @Column({ primaryKey: true, autoIncrement: true })
+  id: number;
+
+  @Column
+  name: string;
+
+  @Column
```

Comment:
For timestamp related columns sequelize has other decorators 
@CreatedAt
@UpdatedAt
and so on we can use those instead of @Column decorators.
This way sequelize will automatically handle timestamps and do a few things like -
1. Add a `deleatedAt is null` check on all find queries.
2. Automatically populated `createdAt` when creating a new entity.
3. Automatically update `updatedAt` when an entity is updated